doc/manual/source/index.rst

```diff
@@ -10,5 +10,15 @@ The manual goes here ...
    :name: toc-reference
    
    man/dnst
+   man/dnst-key2ds
+   man/dnst-keygen
+   man/dnst-notify
    man/dnst-nsec3-hash
-
+   man/dnst-signzone
+   man/dnst-update
+   man/ldns-key2ds
+   man/ldns-keygen
+   man/ldns-notify
+   man/ldns-nsec3-hash
+   man/ldns-signzone
+   man/ldns-update
```

Comment:
I think these should be in a separate `toctree`.

Reply:
I made a quick change to it, but am unsure about it. When in the dnst tree, it hides the ldns compat tools. So the ldns reference is only visible when in the home/startpage. Please have a look when I push it, if that's ok behaviour. Otherwise I'll look up how to show multiple toctrees next week.

Also how should the ldns reference be called? "LDNS Tools Reference" or "LDNS Compatibility Reference" or something else?

Reply:
I like the "tools" version you've got now!